src/content/reference/cluster-upgrades/index.md

```diff
@@ -0,0 +1,190 @@
+This leads to removeal and reacreation of the Pods.
+
+## How to prepare your workloads {#recommendations}
+
+The following recommendations should help you harden your workload deployments for a tpyical upgrade process. Furthermore, they also make your workloads more resilient against unpredicted failures, high load, and resource pressure in your cluster.
+
+### Scale up and distribute workloads
+
+Make sure you have 2 or more replicas for all your deployments. For critical components you might want to go with more than just the bare minimum of 2.
+
+You can adjust this depending on your environments, e.g. running 2-3 replicas in `DEV`, and 5 or more in `PROD`.
+
+In case you are using Horizontal Pod Autoscaler, above recommendations should depict your minimum replica setting.
+
+Additionally, you should make sure your replicas do not land on the same node using [inter-pod anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity-beta-feature).
+
+We recommend always using soft anti-affinity to avoid exclusivity of nodes. However, this can in cases to less distirbuted workloads on resource pressure or node failure, which will need a rescheduling to be balanced again.
+
+For such rescheduling or rebalancing you should have a look at the incubation project called [descheduler](https://github.com/kubernetes-incubator/descheduler) and evaluate its use in your clusters. It has settings for avoiding affinities, but also for rebalancing clusters with under-ultilized nodes.
+
+### Manage disruption budgets
+
+Configure [PodDisruptionBudgets](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) for all your deployments. This tells Kubernetes to keep a minimum amount of Pods running at all times and is respected by the draining/eviction mechanisms during upgrades.
+
+### Make rescheduling graceful
+
+To make rescheduling during upgrades of both the cluster as well as your own deployments more graceful you should take care of a few settings on your Pods. However, these settings might not be enough, your application must be able to handle standard termination signals and have a procedure for gracefully shutting down. Further, it needs to expose some sort of liveness and/or readiness endpoints for Kubernetes to be able to probe it.
+
+1. Have well implemented [liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).
+   Often, a container being alive does not mean it is ready to recieve traffic. By differniating between liveness and readiness you can not only control when traffic gets routed to a fresh container, you can also influence graceful termination of your container.
+2. Set proper values for initialDelaySeconds (related to previous point).
+3. Set proper values for termincationGracePeriod to give your Pod time to gracefully shut down.
```

Comment:
I would highlight specific terms like `termincationGracePeriod`.

Reply:
I would put them in tics like we usually do for such things.

also should be `terminationGracePeriod`